nanofi/tests/CLogAggregatorTests.cpp

```diff
@@ -31,18 +31,19 @@
 
 #include "CTestsBase.h"
 
-void test_lists_equal(token_list * tknlist, const std::vector<std::string>& sv) {
+void test_lists_equal(const token_list * tknlist, const std::vector<std::string>& sv) {
     REQUIRE(tknlist != NULL);
     if (sv.empty()) {
```

Comment:
```suggestion
    REQUIRE(tknlist->size == sv.size());
    token_node *node = tknlist->head;
    for (const auto& s : sv) {
      REQUIRE(node);
      REQUIRE(strcmp(s.c_str(), node->data) == 0);
      node = node->next;
    }
    REQUIRE(node == nullptr);
```

Reply:
good idea, changed it in [e010b24](https://github.com/apache/nifi-minifi-cpp/pull/1035/commits/e010b24559ee441d528884f70e05eb68aec08400)